setup.py

```diff
@@ -12,7 +12,10 @@
     author="Algorand",
     author_email="pypiservice@algorand.com",
     python_requires=">=3.9",
-    install_requires=["py-algorand-sdk>=2.0.0", "tabulate==0.9.0"],
+    install_requires=[
+        "py-algorand-sdk@git+https://github.com/algorand/py-algorand-sdk@better-exports",
```

Comment:
This py-algorand-sdk PR will hopefully get support and be merged soon: https://github.com/algorand/py-algorand-sdk/pull/435